action/DetectObjects.action

```diff
@@ -0,0 +1,6 @@
+# Empty Goal
+---
+# Result
+mas_perception_msgs/ObjectList objects      # planes containing detected objects
```

Comment:
you can probably remove the comment here

Reply:
Right, copy-paste left-over. I force pushed in order not to clutter the commit history.